### PR TITLE
Add releases for OICD bugfix

### PR DESCRIPTION
**Pull request by MarcelMue**

### CODE REVIEW

**Comment by MarcelMue** [comment]
Operators are tagged and deployed - I will take care of merging this actual release on Monday.

**Comment by MarcelMue** [comment]
PTAL these releases are necessary because of a bug in OIDC settings for talos. Is there a need to announce small patches like this through slack?

**Comment by omegas27** [comment]
As long as we do not automatically deploy patch releases to customer clusters, I think we have to :( They would not understand where this release is coming from otherwise

File: release-notes/kvm/v11.3.1.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## :zap: Giant Swarm Release 11.3.1 for KVM :zap:
+
+This release fixes a problem that prevented clusters with OIDC settings to work as expected in `11.3.1`.
```
Comment (tfussell) [inline release-notes/kvm/v11.3.1.md:3]:
```suggestion
This release fixes a problem that prevented clusters with OIDC settings to work as expected in `11.3.0`.
```

File: kvm.yaml [hunk 2]
```diff
@@ -3,9 +3,60 @@ kind: Release
 metadata:
   annotations:
     giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
-  name: v11.3.0
+  name: v11.3.1
 spec:
   state: active
+  date: 2020-04-30T12:00:00Z
```
Comment (tfussell) [inline kvm.yaml:9]:
```suggestion
  date: 2020-05-04T12:00:00Z
```

File: kvm.yaml [hunk 3]
```diff
@@ -322,9 +373,60 @@ kind: Release
 metadata:
   annotations:
     giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
-  name: v9.0.1
+  name: v9.0.2
 spec:
   state: active
+  date: 2020-04-30T12:00:00Z
```
Comment (tfussell) [inline kvm.yaml:379]:
```suggestion
  date: 2020-05-04T12:00:00Z
```

**Comment by sslavic** [comment]
README.md wasn't updated with the two releases. Was it intentional?

**Comment by MarcelMue** [comment]
Will need a follow up either way. Wanted to get the release out while upgrades were ongoing to make it more reliable to choose.

**Pull request merged**